app/views/users/ticket_sales.py

```diff
@@ -386,7 +386,7 @@ def download_orders_as_csv(event_id):
     response = make_response(value)
     response.headers['Content-Type'] = 'text/csv'
     response.headers['Content-Disposition'] = \
-        'inline; filename=Orders-%s_%s.csv' % (event.name, event.created_at.strftime("%d-%b-%Y_%H_%M_%S"))
+        'inline; filename= %s.csv' % (event.name)
```

Comment:
@geekyd could you put the event name in `snake_case` and ensure that any other characters other than `A-Z,a-z,0-9` are stripped from the event name before converting to `snake_case`